### PR TITLE
Add date to request

### PR DESCRIPTION
**Pull request by letiesperon**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -10,7 +10,22 @@
...
+document.getElementById("modal-content-"+bookId).style.display = "block" ;
+$('.datepicker').datepicker();
+}
+
+
+
+function hideModal(bookId) {
+document.getElementById("modal-"+bookId).style.display = "none" ;
+document.getElementById("modal-content-"+bookId).style.display = "none" ;
+}
```
Comment (matiasdh) [inline app/assets/javascripts/application.js]: Ident this file :)

File: app/assets/stylesheets/requests.scss [hunk 2]
```diff
@@ -12,3 +12,41 @@
...
+width:300px;
+height: 100px;
+padding-top: 30px;
+text-align:center;
+background:#fff;
+z-index:101;
+font-family: 'Lato',  sans-serif;
+font-weight: 800;
+padding-right: 10px;
+}
```
Comment (matiasdh) [inline app/assets/stylesheets/requests.scss]: Fix indentation

File: app/assets/stylesheets/books.scss [hunk 3]
```diff
@@ -117,6 +117,14 @@ h3{
   border-width: 1px;
 }
 
+.book_img{
+  position:relative;
+  margin: auto auto;
+  width: 200px;
+  height: 250px;
+
```
Comment (matiasmansilla1989) [inline app/assets/stylesheets/books.scss]: clean this empty line

File: app/assets/stylesheets/books.scss [hunk 4]
```diff
@@ -236,3 +246,5 @@ hr{
   float: right;
 }
 
+
+
```
Comment (matiasmansilla1989) [inline app/assets/stylesheets/books.scss]: remove these extra lines

File: app/assets/stylesheets/requests.scss [hunk 5]
```diff
@@ -12,3 +12,41 @@
...
+.datepicker{
+  width: 200px;
+  height: 30px;
+}
+
+
+/*Modal*/
+  /*Fondo negrito*/
+.modal {
+display: none;
```
Comment (matiasmansilla1989) [inline app/assets/stylesheets/requests.scss]: indent

File: app/assets/stylesheets/requests.scss [hunk 6]
```diff
@@ -12,3 +12,41 @@
...
+height:100%;
+position:fixed;
+background-color: rgba(1, 1, 1, 0.6);
+top:0;
+left:0;
+z-index:100;
+}
+  /*Contenido*/
+.modal-content {
+position: absolute;
```
Comment (matiasmansilla1989) [inline app/assets/stylesheets/requests.scss]: same as above

File: app/views/books/index.json.jbuilder [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+json.books @books do |book|
+  json.title book.title
+  json.author book.author.name
+  json.year book.year
+  json.ISBN book.ISBN
+
```
Comment (matiasmansilla1989) [inline app/views/books/index.json.jbuilder]: remove this extra line

File: app/views/books/index.json.jbuilder [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
+json.books @books do |book|
+  json.title book.title
+  json.author book.author.name
```
Comment (matiasmansilla1989) [inline app/views/books/index.json.jbuilder]: N + 1 problem here, we can discuss this issue later, or ask Maic or MatiDH

**Pull request merged**